pkg/aws/validations/getRole.go

```diff
@@ -0,0 +1,60 @@
+package validations
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/service/iam"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/openshift/rosa/pkg/aws/tags"
+	semver "github.com/hashicorp/go-version"
+)
+
+func GetRoleName(prefix string, role string) string {
+	name := fmt.Sprintf("%s-%s-Role", prefix, role)
+	if len(name) > 64 {
+		name = name[0:64]
+	}
+	return name
+}
+
+func IsManagedRole(roleTags []*iam.Tag) bool {
+    for _, tag := range roleTags {
+        if aws.StringValue(tag.Key) == tags.ManagedPolicies && aws.StringValue(tag.Value) == "true" {
+            return true
+        }
+    }
+
+    return false
+}
+
+func HasCompatibleVersionTags(iamTags []*iam.Tag, version string) (bool, error) {
```

Comment:
Also here, maybe we need more generic file names.
As this function is not related to a `getRole` AWS call.

Reply:
@oriAdler what naming do you suggest?

Reply:
Maybe `iam` can work here? as all validations are related to either role or role tags.